### DISCUSSION TITLE
mosdns ios服务的问题

### DISCUSSION THREAD

**Discussion by ypdenet** [IrineSistiana/mosdns#734] [Q&A]
使用openwrt 的mosdns 5的版本 苹果的推送不正常  http://appleid.apple.com/无法访问 5.13 和最新的版本来回切换试了都不行

**Comment by qingtian110** [comment]
我用的 openwrt luci 的 mosdns，默认配置下可以正常解析它 

![image](https://github.com/IrineSistiana/mosdns/assets/87856890/430c8ec4-7dbb-4a00-8c80-453aac844b29)
